woocommerce/woocommerce
# Modify components empty state

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR creates an EmptyState component for multiple uses. This abstraction was meant to be used under the Variations, Attributes, and Custom Fields sections.  

![Screenshot 2024-05-15 at 1 40 13 PM](https://github.com/woocommerce/woocommerce/assets/1314156/65f824bf-3d61-40dd-8fd6-91bcfa148126)

Closes #47293.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the New product editor is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Go to Products > Organization.
3. The `Attributes` section should show the empty state

![Screenshot 2024-05-15 at 1 50 14 PM](https://github.com/woocommerce/woocommerce/assets/1314156/4b25081e-b219-4544-bae6-32cc81e49dbc)

4. Press the `Show custom fields toggle`.
5. The empty state for the `Custom fields` should be visible.

![Screenshot 2024-05-15 at 1 40 13 PM](https://github.com/woocommerce/woocommerce/assets/1314156/65f824bf-3d61-40dd-8fd6-91bcfa148126)

6. Go to the `Variations` tab.
7. The empty state for the Variations should be visible as well.

![Screenshot 2024-05-15 at 1 49 24 PM](https://github.com/woocommerce/woocommerce/assets/1314156/f6b4b374-038b-4179-98a3-8b9e82ad6cc0)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
